### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by dhaiducek**
- Modify release instructions since fast-forwarding should be disabled between releases.

### CODE REVIEW

**Comment by dhaiducek** [comment]
CI is blocked by the v3.14.1 release, to be set up here:
- #61

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.14.0
\ No newline at end of file
+3.16.0
```
Comment (yiraeChristineKim) [inline VERSION:1]: CI Tests failed because v3.16.0 in upstream(OPA) does not exist yet. I guess

Comment (dhaiducek) [reply VERSION:1]: Oh, true. I didn't think about that. I've disabled fast-forwarding for now. I'll set this to 3.15 and we can bump it and then re-enable fast-forwarding when we want to release the operator. That probably makes more sense than trying to guess which version we're going to release.

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by dhaiducek** [comment]
Oops--sorry for the extra pushes! It's ready to review now. 😄

File: docs/releasing.md [hunk 2]
```diff
@@ -1,68 +1,88 @@
...
+
     ```shell
-    git tag -a -m "Release ${RELEASE_VERSION}" ${RELEASE_VERSION} upstream/main
+    RELEASE_VERSION="$(cat VERSION)"
+    git tag -a ${RELEASE_VERSION} -m "${RELEASE_VERSION}" upstream/main
     git push upstream ${RELEASE_VERSION}
     ```
+
+16. Disable [fast-forwarding](https://github.com/stolostron/gatekeeper-operator/actions/workflows/fast_forward.yaml) to
+    prevent unwanted commits from going to the new release.
```
Comment (yiraeChristineKim) [inline docs/releasing.md:88]: How to disable it? Can I comment all fast_forward.yaml?

Comment (dhaiducek) [reply docs/releasing.md:88]: No, you just head over to the action and click "Disable" from the menu:
image

Comment (dhaiducek) [reply docs/releasing.md:88]: (You can see an "Enable" button at the top right now since it's currently disabled.)

**Review by yiraeChristineKim — APPROVED** [review]

File: docs/releasing.md (hunk 2 above)
Comment (yiraeChristineKim) [reply docs/releasing.md:88]: amazing good tip Thanks!!

**Review by yiraeChristineKim — APPROVED** [review]

**Pull request merged**